src/server/game/Globals/ObjectAccessor.cpp

```diff
@@ -81,6 +81,37 @@ HashMapHolder<Player>::MapType const& ObjectAccessor::GetPlayers()
 template class TC_GAME_API HashMapHolder<Player>;
 template class TC_GAME_API HashMapHolder<Transport>;
 
+namespace PlayerNameMapHolder
+{
+typedef std::unordered_map<std::string, Player*> MapType;
+
+MapType& GetContainer()
+{
+    static MapType _objectMap;
```

Comment:
you can move _objectMap directly into the namespace and remove GetContainer() to avoid magic static initialization (it must be threadsafe so compiler ensure that for you)